runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/config/WintertodtNotifyBrazier.java

```diff
@@ -23,19 +22,19 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+
 package net.runelite.client.plugins.wintertodt.config;
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
 @Getter
 @RequiredArgsConstructor
-public enum WintertodtNotifyMode
+public enum WintertodtNotifyBrazier
```

Comment:
The 3 config classes are copies of each other but just with different names

Reply:
You're right, I misunderstood how this worked. Removed the redundant classes.